python/cudf/cudf/utils/dtypes.py

```diff
@@ -578,6 +583,23 @@ def _get_nan_for_dtype(dtype):
         return np.float64("nan")
 
 
+def _decimal_to_int64(decimal: Decimal) -> int:
+    """
+    Convert a decimal.Decimal value to the int64
+    expected by libcudf
```

Comment:
It would be more helpful if the docstring explained what exactly is returned, rather than referring to libcudf. An example would also help make it clear :)